packages/type/package.json

```
@@ -17,9 +15,14 @@
 	"scripts": {
 		"test": "jest",
 		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
-		"build": "rollup -c",
+		"build": "tsc -b src && yarn build-native",
+		"postbuild": "gen-esm-wrapper dist/index.js dist/index.mjs",
+		"prebuild": "yarn configure",
```

```suggestion
		"prebuild": "yarn configure",
		"build": "tsc -b src && yarn build-native",
		"postbuild": "gen-esm-wrapper dist/index.js dist/index.mjs",
```

Nitpick but pre->action->post reads better.